package.json

```diff
@@ -21,6 +21,7 @@
     "@project-serum/anchor": "^0.18.2",
     "@project-serum/serum": "^0.13.38",
     "@solana/web3.js": "^1.30.2",
+    "@solendprotocol/solend-sdk": "^0.1.64",
```

Comment:
is parseReserve just a small snippet or code or is it significant? If former might be good to just port the code directly here, since then we can avoid an extra dependency

also, what is weweth?

Reply:
actually ignore my other message about taking the snippet. I was able to get out the needed files out :)

Reply:
Also weweth is wormhole eth, one of the other eth bridges on solana with
the other being sollet eth. This will likely be the higher liquidity
Eth going forward but they are both fwiw, considered Eth.

Reply:
ty, all good